packages/mobile/src/store/offline-downloads/sagas.ts

```diff
@@ -208,6 +208,7 @@ export function* startSync() {
 }
 
 export function* handleSetReachable() {
+  // Why do we do this?
```

Comment:
@amendelsohn ^

Reply:
Discussed @ daily sync! im currently testing to see if we can remove this without repercussions